src/devtool/solid/App.tsx

```diff
@@ -61,11 +61,34 @@ function App() {
 		}
 	};
 
-	onMount(() => {
+	onMount(async () => {
 		window.handleRunPause = handleRunPause;
 		window.handleStep = handleStep;
 		window.handleReset = handleReset;
 
+		// Wait for WebUI bindings to be ready, then load initial bytecode
+		const loadInitialBytecode = async () => {
+			// Check if WebUI functions are available
+			if (
+				typeof window.load_bytecode === "function" &&
+				typeof window.reset_evm === "function"
+			) {
+				try {
+					await loadBytecode(bytecode());
+					const initialState = await resetEvm();
+					setState(initialState);
+				} catch (err) {
```

Comment:
nit a try catch where we have no idea which method is throwing. Prefer to handle each potential error individually